db/mongodb.go

```diff
@@ -247,3 +258,45 @@ func (m *MongoDB) CreateExtraIndexes(idxs []string) error {
 	slog.Info(fmt.Sprintf("%d index(es) successfully created in the collection %s", len(r), companyTableName))
 	return nil
 }
+
+func (m *MongoDB) Search(q Query) (string, error) {
+	uf := strings.ToUpper(q.Uf)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	coll := m.db.Collection(companyTableName)
+	f := bson.M{"json.uf": uf}
```

Comment:
Seguindo o comentátio anterior

```suggestion
	f := bson.M{"json.uf": q.Uf}
```